data/shared/citizen/scripting/v8/main.js

```diff
@@ -100,31 +101,55 @@ const EXT_LOCALFUNCREF = 11;
 	global.addRawEventHandler = global.addRawEventListener;
 
 	// Client events
-	global.addEventListener = (name, callback, netSafe = false) => {
+	const addEventListener = (name, callback, netSafe = false) => {
 		if (netSafe) {
 			netSafeEventNames.add(name);
 		}
 
 		emitter.on(name, callback);
 	};
-	global.on = global.addEventListener;
+
+	global.addEventListener = addEventListener;
+	global.on = addEventListener;
+
+	// global.RegisterNetEvent = function (name) { netSafeEventNames.add(name); };
+	// global.RegisterServerEvent = global.RegisterNetEvent;
+	global.AddEventHandler = addEventListener;
```

Comment:
Why exactly you need this alias?

Reply:
It was to match lua's API, I'll remove it I guess.